builtin/packer/config_sourcer.go

```diff
@@ -107,7 +107,7 @@ func (cs *ConfigSourcer) read(
 			result.Result = &pb.ConfigSource_Value_Error{
 				Error: status.New(codes.Aborted, err.Error()).Proto(),
 			}
-			return nil, err
+			continue
 		}
 		channelParams.BucketSlug = packerConfig.Bucket
 		channelParams.Slug = packerConfig.Channel
```

Comment:
Five lines down from here (on 118) there's another error that should be appended to result and not returned!

Reply:
ooooo good catch!